samples/cellular/coap_client/sample.yaml

```diff
@@ -3,7 +3,13 @@ sample:
 tests:
   sample.cellular.coap_client:
     build_only: true
+    build_on_all: true
     integration_platforms:
       - nrf9160dk_nrf9160_ns
```

Comment:
Maybe add an entry for the 9161 while you are at it

Reply:
Since I have no possibility to test that, this should be done in a separate PR, as discussed.